enterprise/suppress-user/factory.go

```diff
@@ -21,62 +25,216 @@ type Factory struct {
 	Log             logger.Logger
 }
 
+func (m *Factory) NewSyncerWithBadgerRepo(repoPath string, seederSource func() (io.ReadCloser, error), identity identity.Identifier, pollInterval time.Duration) (*Syncer, Repository, error) {
```

Comment:
Does it need to be exported? 
Please, move this new method to the bottom of the file. `Setup` needs to be in the spotlight.

```suggestion
func (m *Factory) newSyncerWithBadgerRepo(repoPath string, seederSource func() (io.ReadCloser, error), identity identity.Identifier, pollInterval time.Duration) (*Syncer, Repository, error) {
```